source/collections/Queue.ooc

```diff
@@ -79,8 +79,7 @@ VectorQueue: class <T> extends Queue<T> {
 
 	operator [] (index: Int) -> T {
 		version(safe)
-			if (index < -this count || index >= this count)
-				raise("Indexing in get accessor of VectorQueue outside of range.")
+			raise(index < -this count || index >= this count, "Indexing in get accessor of VectorQueue outside of range.")
```

Comment:
I didn't even know you could make those conditional... nice :+1:
Anyway, I think we can put them on one line just as long they don't exceed 200 or so characters.